srml/babe/src/lib.rs

```diff
@@ -56,6 +58,20 @@ impl BabeInherentData for InherentData {
 	}
 }
 
+/// Logs in this module.
+pub type Log<T> = RawLog<T>;
```

Comment:
I am just curious why do we need this type alias?

Reply:
I think impl_outer_log macro use the RawLog definition. And this might be for type inference

Reply:
@pepyakin

> I am just curious why do we need this type alias?

`impl_outer_log!` requires it.